api/v3/openapi.yaml

```diff
@@ -1709,6 +1840,50 @@ components:
             consumers:
               related: 'http://localhost:9391/v3/clusters/cluster-1/consumer-groups/consumer-group-1/consumers'
 
+    GetConsumerGroupLagResponse:
+      description: 'The max and total consumer lag in a consumer group.'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ConsumerGroupLagData'
+          example:
+            kind: 'KafkaConsumerGroupLag'
+            metadata:
+              self: 'http://localhost:9391/v3/clusters/cluster-1/consumer-groups/consumer-group-1/lag'
+              resource_name: 'crn:///kafka=cluster-1/consumer-groups=consumer-group-1/lag'
+            cluster_id: 'cluster-1'
+            consumer_group_id: 'consumer-group-1'
+            max_lag: 100
+            total_lag: 110
+            consumer_id: 'consumer-1'
+            instance_id: 'consumer-instance-1'
+            client_id: 'client-1'
+            max_topic_name: 'topic-1'
+            max_partition_id: 1
+            max_partition:
+              related: 'http://localhost:9391/v3/clusters/cluster-1/topics/topic-1/partitions/1'
+
+    GetConsumerLagResponse:
+      description: 'The consumer lag.'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ConsumerLagData'
+          example:
+            kind: 'KafkaConsumerLag'
+            metadata:
+              self: 'http://localhost:9391/v3/clusters/cluster-1/consumer-groups/consumer-group-1/consumers/consumer-1/lag'
+              resource_name: 'crn:///kafka=cluster-1/consumer-group=consumer-group-1/consumer=consumer-1/lag'
+            cluster_id: 'cluster-1'
+            consumer_group_id: 'consumer-group-1'
+            consumer_id: 'consumer-1'
+            instance_id: 'consumer-instance-1'
+            client_id: 'client-1'
+            topic_name: 'topic-1'
+            partition_id: 1
+            partition:
+              related: 'http://localhost:9391/v3/clusters/cluster-1/topics/topic-1/partitions/1'
+
```

Comment:
```
GetConsumerLagResponse:
      description: 'The consumer lag.'
      content:
        application/json:
          schema:
            $ref: '#/components/schemas/ConsumerLagData'
          example:
            kind: 'KafkaConsumerLag'
            metadata:
              self: 'http://localhost:9391/v3/clusters/cluster-1/consumer-groups/consumer-group-1/consumers/consumer-1/lag'
              resource_name: 'crn:///kafka=cluster-1/consumer-group=consumer-group-1/consumer=consumer-1/lag'
            cluster_id: 'cluster-1'
            consumer_group_id: 'consumer-group-1'
            consumer_id: 'consumer-1'
            instance_id: 'consumer-instance-1'
            client_id: 'client-1'
            partition_lag:
              - kind: 'PartitionLag'
                current_offset: 1
                log_end_offset: 101
                lag: 100
                topic_name: 'topic-1'
                partition_id: 1
                partition:
                  related: 'http://localhost:9391/v3/clusters/cluster-1/topics/topic-1/partitions/1'
              - kind: 'PartitionLag'
                current_offset: 1
                log_end_offset: 10
                lag: 10
                topic_name: 'topic-1'
                partition_id: 2
                partition:
                  related: 'http://localhost:9391/v3/clusters/cluster-1/topics/topic-1/partitions/2'
```
@rigelbm If I want to include an array of partition lags, would it look something like the above?